Nexus Mods/dist/metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+  "author": {
+    "name": "Bob Ross",
+    "id": "104726244964196352"
+  },
+  "url": "nexusmods.com",
+  "description": {
+    "en": "We support modding for all PC games. If you can mod it, we'll host it."
+  },
+  "service": "Nexus Mods",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/An5OwGg.png",
+  "thumbnail": "https://i.imgur.com/RCC92Ny.png",
+  "color": "#DA8E35",
+  "tags": [
+    "nexus",
+    "mods",
+    "modding",
+    "nexusmods",
+    "vortex",
+    "modmanager",
+    "nexusmodmanager"
+  ],
+  "category": "game",
```

Comment:
Needs to be `games`

Reply:
Will fix. Should probably update the json generator to reflect this as well, seems to generate with `game`.